Documentation/user-guides/getting-started.md

```diff
@@ -229,6 +229,171 @@ By default, Prometheus will only pick up ServiceMonitors from the current
 namespace. To select ServiceMonitors from other namespaces, you can update the
 `spec.serviceMonitorNamespaceSelector` field of the Prometheus resource.
 
+## Deploying Prometheus in Agent mode
+
+Since Prometheus-Operator `v0.64.0` it is also possible to run Prometheus in Agent mode.
+As mentioned in [Prometheus's blog](https://prometheus.io/blog/2021/11/16/agent/), Prometheus Agent
+is a deployment model optimized for environments that expect that all collected data is being
+forwarded to a Long-term storage and local Prometheus don't not need storage at all.
+
+To make it easier to upgrade from `v0.63.0` to `v0.64.0`, the operator is capable of running without
+permissions to the PrometheusAgent CR, but it won't reconcile PrometheusAgent resources if that's the case. So the first
+thing we need to do is make sure Prometheus-Operator has all the permissions necessary.
+
+```yaml mdox-exec="cat example/rbac/prometheus-operator/prometheus-operator-cluster-role.yaml"
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  labels:
+    app.kubernetes.io/component: controller
+    app.kubernetes.io/name: prometheus-operator
+    app.kubernetes.io/version: 0.64.0
+  name: prometheus-operator
+rules:
+- apiGroups:
+  - monitoring.coreos.com
+  resources:
+  - alertmanagers
+  - alertmanagers/finalizers
+  - alertmanagers/status
+  - alertmanagerconfigs
+  - prometheuses
+  - prometheuses/finalizers
+  - prometheuses/status
+  - prometheusagents
+  - prometheusagents/finalizers
+  - prometheusagents/status
+  - thanosrulers
+  - thanosrulers/finalizers
+  - servicemonitors
+  - podmonitors
+  - probes
+  - prometheusrules
+  verbs:
+  - '*'
+- apiGroups:
+  - apps
+  resources:
+  - statefulsets
+  verbs:
+  - '*'
+- apiGroups:
+  - ""
+  resources:
+  - configmaps
+  - secrets
+  verbs:
+  - '*'
+- apiGroups:
+  - ""
+  resources:
+  - pods
+  verbs:
+  - list
+  - delete
+- apiGroups:
+  - ""
+  resources:
+  - services
+  - services/finalizers
+  - endpoints
+  verbs:
+  - get
+  - create
+  - update
+  - delete
+- apiGroups:
+  - ""
+  resources:
+  - nodes
+  verbs:
+  - list
+  - watch
+- apiGroups:
+  - ""
+  resources:
+  - namespaces
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - networking.k8s.io
+  resources:
+  - ingresses
+  verbs:
+  - get
+  - list
+  - watch
+```
+
+Now that we're past that, we just need to deploy the agent similarly to how we deployed Prometheus Server.
```

Comment:
```suggestion
The operator assigns different Kubernetes service accounts to Prometheus and Prometheus Agent. Because of this, it's also necessary to create a new service account for the Agent with the necessary permissions to scrape targets.
```

Reply:
Yes, when we create the Statefulset
Server: https://github.com/prometheus-operator/prometheus-operator/blob/main/pkg/prometheus/server/statefulset.go#L501
Agent: https://github.com/prometheus-operator/prometheus-operator/blob/main/pkg/prometheus/agent/statefulset.go#L400

Reply:
Sorry, I got confused with `Service` my bad :face_palm: here is an updated version
```
Similarly to Prometheus, Prometheus Agent will require permission to scrape targets. Because of this, we will create a new service account for the Agent with the necessary permissions to scrape targets.
```